app/models/company.py

```
@@ -0,0 +1,16 @@
+from sqlalchemy import Column, String
+from sqlalchemy.orm import relationship
+from sqlmodel import Field
```

Ruff (0.8.2)

`sqlmodel.Field` imported but unused

Remove unused import: `sqlmodel.Field`

(F401)